Modules/Core/Common/include/itkProcessObject.h

```diff
@@ -893,7 +893,7 @@ class ITKCommon_EXPORT ProcessObject : public Object
    * updated in derived filters.
    */
   itkGetConstMacro(ThreaderUpdateProgress, bool);
-  itkBooleanMacro(ThreaderUpdateProgress);
+  itkBooleanMacro(ThreaderUpdateProgress)
```

Comment:
Look at the ITK_MACRO_NOOP mechanism.

We want the ; colon here to keep the code consistent.  It may be that the we need to update the itkMacro.h to get rid of these.

Reply:
NOTE:  ALSO ENSURE THAT "ITK_LEGACY_REMOVE" and ITK_FUTURE_LEGACY_REMOVE is turned on when running this flag for testing.

Reply:
>We want the ; colon here to keep the code consistent.

You want the semi colon there? Even though you wouldn't if the macro's equivalent was written out by hand?

Example, `itkBooleanMacro(ThreaderUpdateProgress)` written manually would be:

```
  virtual void ThreaderUpdateProgressOn() { this->SetThreaderUpdateProgress(true); }
  virtual void ThreaderUpdateProgressOff() { this->SetThreaderUpdateProgress(false); }
```

which you wouldn't generally write as:

```
  virtual void ThreaderUpdateProgressOn() { this->SetThreaderUpdateProgress(true); }
  virtual void ThreaderUpdateProgressOff() { this->SetThreaderUpdateProgress(false); };
```

As this macro isn't a function-like macro, it seems odd to end it with a semi.